microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -201,6 +201,9 @@ The response value from `navigator.userAgent` method uses the following format:
 
 Platform identifiers change based on the operating system, and version numbers increment over time. The format is the same as the Chromium user agent with the addition of a new `Edg` token at the end. Microsoft chose the `Edg` token to avoid compatibility issues caused by `Edge` string, which was previously used for the legacy Microsoft Edge browser based on EdgeHTML. The `Edg` token is also consistent with [existing tokens](https://blogs.windows.com/msedgedev/2017/10/05/microsoft-edge-ios-android-developer) used for iOS and Android.
 
+> [!NOTE]
+> Microsoft Edge's major version number will soon be incrementing from two digits, such as `Edg/99`, to three digits as in `Edg/100`. Site owners should make sure their User-Agent parsing logic is robust and works as expected.

```

Comment:
```suggestion
> [!NOTE]
> Microsoft Edge's major version number will soon be incrementing from two digits, such as `Edg/99`, to three digits as in `Edg/100`. Site owners should make sure their User-Agent parsing logic is robust and works as expected.

```
Most types of Alerts are deprecated, because they backfire and prevent people from seeing the "bumped-out-of-article-body" message; they backfire: https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=main#alerts-note-tip-important-caution-warning - "Avoid Notes, Tips, and Important boxes. Readers tend to skip over them. It's better to put that info directly into the article text." Caution and Warning are ok, according to DCG.

Reply:
Maybe better than "Note: "  is a proven big improvement: pls try or consider adding an h3 or h2 label - this is the usual big improvement to make to get rid of Alerts and do better.